apps/ledger-live-desktop/src/renderer/families/ethereum/StakeFlowModal/EthStakingModalBody.tsx

```diff
@@ -36,6 +43,7 @@ export function EthStakingModalBody({
 }: Props) {
   const { t } = useTranslation();
   const history = useHistory();
+  const [doNotShowAgain, setDoNotShowAgain] = useState<boolean>(false);
```

Comment:
just a question.
is the initial state of doNotShowAgain "false"?
should we initialise the state with the value in "global.localStorage" ?

Reply:
There are 2 flows when we show this modal:
1. When the user want specifically trigger ETH stake flow
2. When the user trigger receive flow. In the last step, we are promoting "stake". In that flow we can dismiss the modal (do not show again).

"doNotShowAgain" is the state of the checkbox itself. If the checkbox is selected or not.
"global.localStorage" is used to dismiss that modal when you are in the receive flow.